scipy/spatial/transform/_rigidtransformation.pyx

```diff
@@ -0,0 +1,1899 @@
+    ...     ax.text(*t.translation, name, color="k", va="center", ha="center",
+    ...             bbox={"fc": "w", "alpha": 0.8, "boxstyle": "circle"})
+
+    **Defining Frames**
+
+    Let's work through an example.
+
+    First, define the "world frame" A, also called the "base frame".
+    All frames are the identity transformation from their own perspective.
+
+    >>> t_A = T.identity()
+
+    We will visualize a new frame B in A's coordinate system. So we need to
+    define the transformation that converts coordinates from frame B to frame
+    A (A <- B).
+
+    Physically, let's imagine constructing B from A by:
+
+    1) Rotating A by +90 degrees around its x-axis.
+    2) Translating the rotated frame +2 units in A's -x direction.
+
+    From A's perspective, B is at [-2, 0, 0] and rotated +90 degrees about the
+    x-axis, which is exactly the transform A <- B.
+
+    >>> r_A_B = R.from_euler('xyz', [90, 0, 0], degrees=True)
+    >>> d_A_B = np.array([-2, 0, 0])
+    >>> t_A_B = T.from_rottrans(r_A_B, d_A_B)
+
+    ``t_B_A`` is the inverse, i.e. the transformation B <- A.
+
+    >>> t_B_A = t_A_B.inv()
+
+    Let's plot these frames.
+
+    >>> fig, ax = plt.subplots(subplot_kw={"projection": "3d"})
+    >>> plot_transformed_axes(ax, t_A, name="tA")     # A plotted in A
+    >>> plot_transformed_axes(ax, t_A_B, name="tAB")  # B plotted in A
+    >>> ax.set_title("A, B frames with respect to A")
+    >>> ax.set_aspect("equal")
+    >>> ax.figure.set_size_inches(6, 5)
+    >>> plt.show()
+
+    Now let's visualize a new frame C in B's coordinate system.
+    Let's imagine constructing C from B by:
+
+    1) Translating B by 2 units in its +z direction.
+    2) Rotating B by +30 degrees around its z-axis.
+
+    >>> d_B_C = np.array([0, 0, 2])
+    >>> r_B_C = R.from_euler('xyz', [0, 0, 30], degrees=True)
+    >>> t_B_C = T.from_rottrans(r_B_C, d_B_C)
+    >>> t_C_B = t_B_C.inv()
+
+    In order to plot these frames from a consistent perspective, we need to
+    calculate the transformation between A and C. Note that we do not make this
+    transformation directly, but instead compose intermediate transformations
+    that let us get from A to C:
+
+    >>> t_C_A = t_C_B * t_B_A  # C <- B <- A
+    >>> t_A_C = t_C_A.inv()
+
+    Now we can plot these three frames from A's perspective.
+
+    >>> fig, ax = plt.subplots(subplot_kw={"projection": "3d"})
+    >>> plot_transformed_axes(ax, t_A, name="tA")     # A plotted in A
+    >>> plot_transformed_axes(ax, t_A_B, name="tAB")  # B plotted in A
+    >>> plot_transformed_axes(ax, t_A_C, name="tAC")  # C plotted in A
+    >>> ax.set_title("A, B, C frames with respect to A")
+    >>> ax.set_aspect("equal")
+    >>> ax.figure.set_size_inches(6, 5)
+    >>> plt.show()
+
+    **Transforming Vectors**
+
+    Let's transform a vector from A, to B and C. We define a point in A and
+    use the transformations we defined to transform it to coordinates in B and
+    C:
```

Comment:
I think inverse transforms can be introduced here.

Perhaps explain `inverse` argument of `apply` too, but not necessary.